docs/advanced/chaos-experimentation.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Chaos Experimentation Framework
+{{ .EditURL }}
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+A framework to perform Chaos experiments built on top of [EnvoyProxy](https://www.envoyproxy.io/docs/envoy/latest/)
+
+<img alt="Clutch Component Architecture" src={useBaseUrl('img/docs/chaos-experimentation.png')} width="75%" />
+
+Chaos Experimentation Framework consists of few parts - frontend, a backend server and a xDS management server. 
+
+The Frontend uses [Clutch's core frontend](https://clutch.sh/docs/development/frontend). It can be customized by using the frontend config. 
+
+The Backend server is responsible for all the CRUD operations of the experiments. It stores experiments in its tables in the Postgres database. 
+
+The xDS management server uses [go-control-plane](http://github.com/envoyproxy/go-control-plane) library and serves two Envoy APIs - [RTDS](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/operations/dynamic_configuration#rtds) and [ECDS](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/operations/dynamic_configuration#ecds). Either of these two APIs can be used to perform fault injection tests. With RTDS, you can make changes to runtimes specific to faults whereas with ECDS you can make changes to the entire fault filter to perform any custom experiments.
```

Comment:
Could we add a bit on why we support both? Maybe something like - functionality wise they are basically 1:1 it boils down to what mechanism - RTDS or ECDS - you use at your company or something like this

Reply:
Also, redis fault injection tests will be possible to run with RTDS only (not sure whether we need to include this information).

Reply:
I have mentioned that both ECDS and RTDS can be used to perform fault injection along with the difference capability wrt fault injection. 

I agree on adding that Redis experiments can only be performed with RTDS. I will add it to the docs